data/hai/midas.txt

```diff
@@ -0,0 +1,1010 @@
+			choice
+				`	"May I wear your colors as well?"`
+					goto wearcolors
+				`	"How can I restore my reputation with the Unfettered? Attacking the previous flagship seem to have upset them."`
+					to display
+						"reputation: Hai (Unfettered)" < 0
+				`	"You did not even ask me if I wanted to be a part of it."`
+					goto asked
+				`	(End the connection.)`
+					goto endco
+			`	"They will probably forget about it if you sell them a Jump Drive."`
+			choice
+				`	"May I wear your colors as well?"`
+				`	(End the connection.)`
+					goto endco
+			label wearcolors
+			action
+				event "joined midas"
+			`	"You may paint your ship however you like." They then end the connection.`
+				goto endco
+			label asked
+			`	"That's right. And I won't." They cut the transmission.`
+			label endco
+			`	"Now you should leave the system. I will have to get some things straight with the others before they stop shooting you permanently, some still consider you an enemy."`
+			action
+				"reputation: Midas" >?= 100
+				"salary: Midas" = 5000
+				set "midas: took out the Shadow"
+				"midas: proven worthy" ++
+				clear "midas: wronged"
+				log "Took out the leader of the Midas organization that manages the Gold Bars. They did not reveal much, only that they do not wish to fight me anymore."
+	npc
+		system "Wah Yoot"
+		government "Shadow"
+		personality heroic staying nemesis quiet
+		ship "Tick" "Hell Unleashed"
+		ship "Tick" "Fire Incarnate"
+		ship "Far Lek 14" "Phoenix"
+	npc
+		system "Wah Yoot"
+		government "Shadow"
+		personality heroic staying nemesis quiet
+		ship "Sea Scorpion" "Thorn wishes Talon"
+		to spawn
+			or
+				not "midas: talon assassination"
+				not "midas: thorn assassination"
+		on kill
+			set "midas: thorn assassination"
+			set "midas: talon assassination"
+	npc kill
+		system "Wah Yoot"
+		government "Shadow"
+		personality disables plunders harvests staying quiet
+		ship "Marauder Firebird (Bars)" "Dark Angel"
+		to spawn
+			not "midas: angel assassination"
+		on kill
+			set "midas: angel assassination"
+	npc
+		system "Wah Yoot"
+		government "Midas"
+		personality heroic staying nemesis vindictive
+		to despawn
+			has "salary: Midas"
+		fleet "Marauders" 8
+	on enter "Wah Yoot"
+		dialog `As you enter Wah Yoot you spot a fleet of mercenaries, larger than anything you've seen from them before. The captain of the fleet hails you. "Nemo Me Impune Lacessit, <first> <last>. You have killed our own. We have come to bring justice upon you, whatever the cost. There will be no negotiations, no forgiveness so long as I draw breath! I am the Shadow of Mintaka, and never again will you see the light."`
+
+
+
+ship "Corvette" "Corvette (Harpy)"
+	outfits
+		"Hai Tracker Pod" 4
+		"Hai Tracker" 112
+		"Heavy Anti-Missile Turret" 2
+		"Fission Reactor"
+		"LP036a Battery Pack"
+		"D94-YV Shield Generator"
+		"Cooling Ducts"
+		"Bunk Room"
+		"Pulse Rifle" 3
+		"Brig" 2
```

Comment:
Two Brigs? 
```suggestion
		"Brig" 1
```

Reply:
yeah, many slaves

Reply:
In the game, one is enough, but lorewise you’re right. This is resolved.